extensions/resource-deployment/notebooks/bdc/2019/deploy-bdc-existing-aro.ipynb

```diff
@@ -0,0 +1,350 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "python3",
+            "display_name": "Python 3"
+        },
+        "language_info": {
+            "name": "python",
+            "version": "3.6.6",
+            "mimetype": "text/x-python",
+            "codemirror_mode": {
+                "name": "ipython",
+                "version": 3
+            },
+            "pygments_lexer": "ipython3",
+            "nbconvert_exporter": "python",
+            "file_extension": ".py"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "![Microsoft](https://raw.githubusercontent.com/microsoft/azuredatastudio/master/src/sql/media/microsoft-small-logo.png)\n",
+                " \n",
+                "## Deploy SQL Server 2019 Big Data Cluster on an existing Azure Red Hat OpenShift cluster\n",
+                " \n",
+                "This notebook walks through the process of deploying a <a href=\"https://docs.microsoft.com/sql/big-data-cluster/big-data-cluster-overview?view=sqlallproducts-allversions\">SQL Server 2019 Big Data Cluster</a> on an existing Azure Red Hat OpenShift cluster.\n",
+                " \n",
+                "* Follow the instructions in the **Prerequisites** cell to install the tools if not already installed.\n",
+                "* The **Required information** will check and prompt you for password if it is not set in the environment variable. The password can be used to access the cluster controller, SQL Server, and Knox.\n",
+                "\n",
+                "<span style=\"color:red\"><font size=\"3\">Please press the \"Run Cells\" button to run the notebook</font></span>"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "23954d96-3932-4a8e-ab73-da605f99b1a4"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "### **Prerequisites** \n",
+                "Ensure the following tools are installed and added to PATH before proceeding.\n",
+                " \n",
+                "|Tools|Description|Installation|\n",
+                "|---|---|---|\n",
+                "|kubectl | Command-line tool for monitoring the underlying Kuberentes cluster | [Installation](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-using-native-package-management) |\n",
```

Comment:
Kubernetes 

Looks like this is misspelled in the other existing notebooks too